### DISCUSSION TITLE
字母的空心字体不好看

### DISCUSSION THREAD

**Discussion by JiaxuanGood** [obster-y/XJTU-thesis#130] [General]
指示符号用空心体的大写字母I来表示（用\mathbb{I}实现），效果很不好看：
![空心错的](https://github.com/user-attachments/assets/448a829e-00f8-4f6c-bf2d-1abae1cc871e)
word或者以前用的LaTeX是这样的效果：
![空心对的](https://github.com/user-attachments/assets/4ac27eda-5352-461f-a104-18742fcd9e70)

**Comment by obster-y** [comment]
1. 这是字体的问题，你觉得「很不好看」，可以自己去换
2. 如果你发这条是想知道怎么换，你要表示你的意图
3. 如果你发这条是想吐槽，那么我的回复是「哦，关我屁事」

**Reply by JiaxuanGood** [reply]
我用了模版给的默认字体，没敢随便换。
我帖两个对比图上来就是为了解决问题，上图的空心I明显长得像个矩形（看着就不像函数），完全不同于下图的空心I（一看就是个函数）。我又不能直接说模版的空心I是错的，因为他确实实现了空心I。我的问题就是怎么实现下图那种样子吗？

**Reply by obster-y** [reply]
1. 如果试一试 `\mathbb{ABCEDFGHIJKLMN}` 就会发现这就是字体本身的设计，`\mathbb` 是为了仿照黑板上的粗体，bb=blackboard bold
3. 这只是一个字体，是你非要用它做函数，然后觉得它不像，完全可以换别的比如 `\mathcal`，一个不是众所周知的定义用的符号通常可以自主决定的吧
5. 如果你想要下面的效果，在导言区插入 `\setmathfont[range=\mathbb]{TeX Gyre Bonum Math}` 可以实现
